packages/react-scripts/config/botifyConfig.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+const autoprefixer = require('autoprefixer');
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+
+const getSCSSLoaderConfig = isDev => {
+  const loaders = [
+    {
+      loader: require.resolve('css-loader'),
+      options: Object.assign({}, { minimize: !isDev }, { importLoaders: 1 }),
+    },
+    {
+      loader: require.resolve('postcss-loader'),
+      options: {
+        // Necessary for external CSS imports to work
+        // https://github.com/facebookincubator/create-react-app/issues/2677
+        ident: 'postcss',
+        plugins: () => [
+          require('postcss-flexbugs-fixes'),
+          autoprefixer({
+            browsers: [
+              '>1%',
+              'last 4 versions',
+              'Firefox ESR',
+              'not ie < 9', // React doesn't support IE8 anyway
+            ],
+            flexbox: 'no-2009',
+          }),
+        ],
+      },
+    },
+    require.resolve('sass-loader'),
+  ];
+
+  if (isDev) {
+    return {
+      test: /\.scss$/,
+      use: [require.resolve('style-loader'), ...loaders],
+    };
+  }
+
+  return {
+    test: /\.scss$/,
+    loader: ExtractTextPlugin.extract({
+      fallback: require.resolve('style-loader'),
+      use: loaders,
+    }),
+  };
+};
+
+module.exports = ({ isDev = true } = {}) => ({
```

Comment:
Is there no way to infer isDev from env variables ? 
`const isDev = NODE_ENV !== 'production'` no ?

Reply:
I think that we can, let me try this.

Reply:
I don't think that NODE_ENV will be set to production here, except if we set it ourself

Reply:
/react-scripts/config/env.js file
![image](https://user-images.githubusercontent.com/14129459/30970669-fe663b9e-a465-11e7-8519-9a0375556934.png)
I'll try and let you updated